src/plugins/fakeProfileThemes/index.tsx

```diff
@@ -57,7 +57,7 @@ function decode(bio: string): Array<number> | null {
     if (bio == null) return null;
 
     const colorString = bio.match(
-        /\u{e005b}\u{e0023}([\u{e0061}-\u{e0066}\u{e0041}-\u{e0046}\u{e0030}-\u{e0039}]+?)\u{e002c}\u{e0023}([\u{e0061}-\u{e0066}\u{e0041}-\u{e0046}\u{e0030}-\u{e0039}]+?)\u{e005d}/u,
+        /\u{e005b}\u{e0023}([\u{e0061}-\u{e0066}\u{e0041}-\u{e0046}\u{e0030}-\u{e0039}]{6})\u{e002c}\u{e0023}([\u{e0061}-\u{e0066}\u{e0041}-\u{e0046}\u{e0030}-\u{e0039}]{6})\u{e005d}/u,
```

Comment:
It has to be `{0,6}` to maintain compatibility with similar plugins for other client mods.
```suggestion
        /\u{e005b}\u{e0023}([\u{e0061}-\u{e0066}\u{e0041}-\u{e0046}\u{e0030}-\u{e0039}]{0,6})\u{e002c}\u{e0023}([\u{e0061}-\u{e0066}\u{e0041}-\u{e0046}\u{e0030}-\u{e0039}]{0,6})\u{e005d}/u,
```

Reply:
My bad, I meant `{1,6}`.

Reply:
But the current code doesn't support non 6 char hex codes. #fff will not be interpreted as white by the current code, but rather as #000fff, which is blue. Therefore I don't see the need for allowing shorter codes

Reply:
It does not deal with CSS colors. The colors always get passed to encode as 24-bit colors, so that is the intended behaviour. If you force it to only accept 6 characters, you will break compatibility with other plugins for no reason.

Reply:
Sorry, that's not how I intended to come across.